applications/asset_tracker_v2/src/events/data_module_event.h

```diff
@@ -17,6 +17,10 @@
 #include <app_event_manager_profiler_tracer.h>
 #include "cloud/cloud_codec/cloud_codec.h"
 
+#if defined(CONFIG_LWM2M)
+#include <zephyr/net/lwm2m.h>
+#endif
+
```

Comment:
Need to have the dummy header version too?

Reply:
Done